src/pages/Insights/InsightsPage.module.scss

```diff
@@ -25,15 +25,16 @@
 }
 
 .mainSection {
-  padding: 0 16px;
+  padding: 0;
 }
 
 .newSignal {
-  width: 100%;
   margin-bottom: 12px;
   display: flex;
   align-items: center;
   justify-content: center;
+  margin-left: 16px;
+  margin-right: 16px;
```

Comment:
![image](https://user-images.githubusercontent.com/25135650/56277349-da363c00-610c-11e9-86d1-d195c11310ad.png)
Misalignment. Here how it was before the change and in the design mock-up
![image](https://user-images.githubusercontent.com/25135650/56277406-f2a65680-610c-11e9-83ab-4b7e65993923.png)